specification/monitor/resource-manager/Microsoft.Monitor/stable/2023-04-03/examples/AzureMonitorWorkspacesCreate.json

```diff
@@ -0,0 +1,90 @@
+    "azureMonitorWorkspaceName": "myAzureMonitorWorkspace",
+    "api-version": "2023-04-03",
+    "azureMonitorWorkspaceProperties": {
+      "location": "eastus"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/myResourceGroup/providers/Microsoft.Monitor/accounts/myAzureMonitorWorkspace",
+        "name": "myAzureMonitorWorkspace",
+        "type": "Microsoft.Monitor/accounts",
+        "location": "eastus",
+        "tags": {
+          "tag1": "A",
+          "tag2": "B"
+        },
+        "properties": {
+          "accountId": "2df515bf-c3ce-4920-84d4-1d9d16542d9f",
+          "metrics": {
+            "prometheusQueryEndpoint": "https://myAzureMonitorWorkspace-v8hx.eastus.prometheus.monitor.azure.com",
+            "internalId": "mac_2df515bf-c3ce-4920-84d4-1d9d16542d9f"
+          },
+          "provisioningState": "Succeeded",
+          "defaultIngestionSettings": {
+            "dataCollectionRuleResourceId": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/MA_myAzureMonitorWorkspace_eastus_managed/providers/Microsoft.Insights/dataCollectionRules/myAzureMonitorWorkspace",
+            "dataCollectionEndpointResourceId": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/MA_myAzureMonitorWorkspace_eastus_managed/providers/Microsoft.Insights/dataCollectionEndpoints/myAzureMonitorWorkspace"
+          },
+          "privateEndpointConnections": [
+            {
+              "id": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/myResourceGroup/providers/Microsoft.Monitor/accounts/myAzureMonitorWorkspace/privateEndpointConnections/myPrivateEndpointConnection",
+              "name": "myPrivateEndpointConnection",
+              "type": "Microsoft.Monitor/accounts/privateEndpointConnections",
+              "properties": {
+                "provisioningState": "Succeeded",
+                "privateEndpoint": {
+                  "id": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/myResourceGroup/providers/Microsoft.Network/privateEndpoints/myPrivateEndpointConnection"
+                },
+                "groupIds": [
+                  "prometheusMetrics"
+                ],
+                "privateLinkServiceConnectionState": {
+                  "status": "Approved",
+                  "description": "Auto-approved",
+                  "actionsRequired": "None"
+                }
+              }
+            }
+          ],
+          "publicNetworkAccess": "Enabled"
+        },
+        "systemData": {
+          "createdBy": "user1",
+          "createdByType": "User",
+          "createdAt": "2021-04-01T12:34:56.1234567Z",
+          "lastModifiedBy": "user2",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2021-04-02T12:34:56.1234567Z"
+        },
+        "etag": "070057da-0000-0000-0000-5ba70d6c0000"
+      }
+    },
+    "201": {
```

Comment:
The API is changing to asynchronous soon. 201 status is fine, but the response body is probably going to be empty.

Reply:
Updated

Reply:
We've decided to leave PUT synchronous, so it's going to be only 200 (no 201 or 202).